docs/source/llms/llm-evaluate/index.rst

```diff
@@ -403,14 +403,15 @@ Prepare Your LLM for Evaluating
 
 In order to evaluate your LLM with ``mlflow.evaluate()``, your LLM has to be one of the following type:
 
-1. A :py:func:`mlflow.pyfunc.PyFuncModel` instance or a URI pointing to a logged `mlflow.pyfunc.PyFuncModel` model. In
+1. A :py:func:`mlflow.pyfunc.PyFuncModel` instance or a URI pointing to a logged ``mlflow.pyfunc.PyFuncModel`` model. In
    general we call that MLflow model. The 
 2. A python function that takes in string inputs and outputs a single string. Your callable must match the signature of 
-   :py:func:`mlflow.pyfunc.PyFuncModel.predict` (without `params` argument), briefly it should:
+   :py:func:`mlflow.pyfunc.PyFuncModel.predict` (without ``params`` argument), briefly it should:
 
    * Has ``data`` as the only argument, which can be a ``pandas.Dataframe``, ``numpy.ndarray``, python list, dictionary or scipy matrix.
    * Returns one of ``pandas.DataFrame``, ``pandas.Series``, ``numpy.ndarray`` or list. 
-3. Set ``model=None``, and put model outputs in `data`. Only applicable when the data is a Pandas dataframe.
+3. An MLflow Deployment endpoint URI pointing to a local `MLflow Deployment Server <../deployments/index.html>`_ or `Databricks Foundation Models <https://docs.databricks.com/en/machine-learning/foundation-models/index.html>`_. 
```

Comment:
```suggestion
3. An MLflow Deployments endpoint URI pointing to a local `MLflow Deployment Server <../deployments/index.html>`_ or `Databricks Foundation Models <https://docs.databricks.com/en/machine-learning/foundation-models/index.html>`_. 
```

Reply:
@B-Step62 MLflow Deployments also supports Databricks External Models. Can we add that?